### PR TITLE
 config controller code optimization

### PR DESCRIPTION
**Pull request by Chaos-coder**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

 code optimization config

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Review by chuntaojun — APPROVED** [review]

File: config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java [hunk 1]
```diff
@@ -118,24 +118,27 @@
...
+                                 @RequestParam(value = "desc", required = false) String desc,
+                                 @RequestParam(value = "use", required = false) String use,
+                                 @RequestParam(value = "effect", required = false) String effect,
+                                 @RequestParam(value = "type", required = false) String type,
+                                 @RequestParam(value = "schema", required = false) String schema) throws NacosException {
         
         final String srcIp = RequestUtil.getRemoteIp(request);
         final String requestIpApp = RequestUtil.getAppName(request);
+
+        String srcUser = RequestUtil.getSrcUserName(request);
```
Comment (li-xiao-shuang) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java:136]: 这个在下面是有赋值的

Comment (li-xiao-shuang) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java:136]: srcUser 下面有赋值 这一行是不是有点重复？

Comment (Chaos-coder) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java:136]: 是的 被另一个pr修改过了

Comment (li-xiao-shuang) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java:136]: > 是的 被另一个pr修改过了

原来是这样  那你可以在这个pr中去掉这一行

Comment (Chaos-coder) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java]: > > 是的 被另一个pr修改过了
> 
> 原来是这样 那你可以在这个pr中去掉这一行

done

**Pull request closed without merging**